src/elm/arithmetic.js

```diff
@@ -23,6 +24,8 @@ class Add extends Expression {
     const sum = args.reduce((x, y) => {
```

Comment:
It used to be that `Add` (and some of the other basic math operations) supported a variable number of arguments, hence the use of `reduce`.  But now, `Add` is a `BinaryExpression` -- meaning it only accepts two arguments.  So if it simplifies things, just be aware that you can decide not to use reduce anymore if you want.

Reply:
Up to you though.